src/transformState.js

```diff
@@ -4,8 +4,27 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+// eslint-disable-next-line no-shadow
 function transformState(state, actions) {
-  // write code here
+  for (const i of actions) {
```

Comment:
Let's create meaningful names for variables
```suggestion
  for (const action of actions) {
```